src/styles/utils/_functions.scss

```diff
@@ -0,0 +1,18 @@
+@for $date from 1 through 31 {
```

Comment:
1. if it's a file for functions you shouldn't add here something else
2. you can use loops inside the calendar.scss, put it in the separate file - not a good idea
watch this lesson:
![image](https://user-images.githubusercontent.com/87998640/221897630-79712116-29d8-491b-8890-e250b2303503.png)